docs/source/array-api.rst

```diff
@@ -90,6 +90,7 @@ Top level user functions:
    isclose
    iscomplex
    isfinite
+   isin
```

Comment:
Missed that we didn't have an `autofunction` line with `isin`. Fixing with PR ( https://github.com/dask/dask/pull/3364 ).